### ISSUE TITLE
It seems that Cirrus-Ci is broken

### ISSUE THREAD

**Issue by tibeer** [osism/k8s-capi-images#138]
New images are not built. Need to investigate. Closes SovereignCloudStack/k8s-cluster-api-provider#248

**Comment by berendt** [comment]
```
�[0;32m    qemu: fatal: [default]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: Unable to negotiate with 127.0.0.1 port 42009: no matching host key type found. Their offer: ssh-rsa", "unreachable": true}�[0m
```

**Comment by berendt** [comment]
Fixed with 50a1e8853bae46db557e78873302b8119ed73b8d
